### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Immortalin**
`worker/3` has been deprecated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,8 @@ run `mix guardian.db.gen.migration` to generate a migration.
 To sweep expired tokens from your db you should add `Guardian.DB.Token.SweeperServer` to your supervision tree.
 
 ```elixir
-  worker(Guardian.DB.Token.SweeperServer, [])
+  %{id: :guardian_db_token_sweeper,
```
Comment (doomspork) [inline README.md]: @Immortalin it looks like there's an even simpler option for this and we should probably use that: https://hexdocs.pm/elixir/Supervisor.html#module-child_spec-1

```suggestion
children = [
  {Guardian.DB.Token.SweeperServer, []}
]
```

**Review by doomspork — CHANGES_REQUESTED** [review]
Just a thought on the format @Immortalin, let me know your thoughts 👍

**Comment by Immortalin** [comment]
@doomspork awesome!

**Comment by doomspork** [comment]
Thank you @Immortalin!

**Pull request merged**